src/React.AspNet/project.json

```diff
@@ -1,43 +1,52 @@
 {
-    "version": "2.3.0-*",
-    "configurations": {
-        "Debug": {
-            "compilationOptions": {
-                "define": [ "DEBUG", "TRACE", "ASPNET5" ]
-            }
-        },
-        "Release": {
-            "compilationOptions": {
-                "define": [ "RELEASE", "TRACE", "ASPNET5" ],
-                "optimize": true,
-                "warningsAsErrors": true
-            }
-        }
+  "version": "2.3.0-*",
+  "authors": [ "Daniel Lo Nigro" ],
+  "packOptions": {
+    "owners": [ "Daniel Lo Nigro" ],
+    "licenseUrl": "https://github.com/reactjs/React.NET#licence",
+    "iconUrl": "http://facebook.github.io/react/img/logo_og.png",
+    "copyright": "Copyright 2014-Present Facebook, Inc",
+    "title": "ReactJS.NET (MVC Core)",
+    "description": "ReactJS and Babel tools for ASP.NET Core, including ASP.NET MVC Core. Please refer to project site (http://reactjs.net/) for full installation instructions, usage examples and sample code",
+    "tags": [ "asp.net", "mvc", "asp", "javascript", "js", "react", "facebook", "reactjs", "vnext" ],
+    "projectUrl": "http://reactjs.net/"
+  },
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG", "TRACE", "ASPNET5" ]
```

Comment:
Ah, good point. Setting preprocess directives was actually broken for ASP.NET 5 projects, so all the conditions instead check for `LEGACYASPNET` and assume a ASP.NET 5 environment if not set (eg. https://github.com/reactjs/React.NET/blob/df0313d652b339b1f6cdbf59811da219611c0973/src/React.AspNet/HtmlHelperExtensions.cs#L13). I guess you can rename this without changing anything then :)